src/timed-stream.ts

```diff
@@ -79,7 +80,6 @@ export class TimedStream extends PassThrough {
         if (options?.transformHook) {
           options?.transformHook(event, _encoding, callback);
         }
-        callback(null, event);
```

Comment:
In the userStreams the callback of the transform function is used in several places and sometimes does not pass an event. So this change relies on the `transformHook` to use the callback in the right places and offers us the flexibility that we need.